release-notes/6.0/preview/api-diff/preview7/.Net/6.0.0-preview7_System.Reflection.md

```diff
@@ -0,0 +1,26 @@
+# System.Reflection
+
+``` diff
+ namespace System.Reflection {
++    public sealed class NullabilityInfo {
++        public NullabilityInfo ElementType { get; }
```

Comment:
NIT: not sure nullability matter for this diff
```suggestion
+        public NullabilityInfo? ElementType { get; }
```
Other than this file LGTM

Reply:
we are not adding any  nullabity changes here.